src/daemon/coordinator.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <daemon/base_daemon.h>
+
+namespace dist_clang {
+namespace daemon {
+
+class Coordinator : public BaseDaemon {
+ public:
+  explicit Coordinator(const proto::Configuration& configuration);
+
+  bool Initialize() override;
+  inline bool UpdateConfiguration(
```

Comment:
Looks like it's a debt. I've tried to use `UpdateConfiguration` from `DoCoordinator`, but that one requires fully specified `Configuration` message, which is not a case for coordinator. I'll remove it.